wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+
+A restriction is a timeout from the community forced upon accounts that exhibit abnormal, suspicious, or rule-breaking behaviour.
+
+When an account is restricted, it is unable to interact with the community until the restriction is lifted. Restricted players can still continue playing and submitting scores, but their profile will be invisible to others.
+
+The following features are disabled for restricted accounts:
+- in-game chat
+- private messaging on the forums
+- posting on forums
+- updating maps or profile details
+- store purchases
+
+## My account has been restricted! What do I do?
+
+If your account has been restricted, do not panic. Please follow these steps:
+- **Wait 24 hours.** Some restrictions are the results of automated processes. As we check every new restriction daily, please wait 24 hours - if your restriction was erroneous, it will be lifted within that time.
+- **Contact our support team.** If your account remains restricted after that time, please send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) to ask about the reasons for it. We'll need at the very least to know your username to be able to check! Remember to send the email from the email address attached to your account so that we can confirm you are the owner.
+
+In the cases where an account becomes restricted for breaking the [rules](/wiki/rules), a cooldown of three months (strictly) will be given before the user is allowed to send an appeal. If the account you asked about is not your first, please check if your original account is restricted or not. If it isn't, you can continue playing on it, but make sure you do not continue accessing your multi-accounts as that might lead to further punishments.
+
+If you have recently been granted an appeal and your account has become restricted again after a play, don't panic. This is something that happens to some appeal returnees. The support team will generally remove those restrictions within 24 hours. Feel free to send us an email or a tweet to catch our attention and be unrestricted faster!
+
+## Why am I restricted?
+
+There aren't that many reasons why an account might be restricted.
+- **It isn't your first account.** Multi-accounting is against the rules. Any accounts other than a person's first will become permanently restricted, but their original one will in most cases be left open and the user is encouraged to continue playing on it. If the multi-accounting is severe or abusive, a silence may be applied to their original account as a warning to cease creating new ones, and a restriction will be placed on the main account if the multi-accounting behaviour continues. If your account is restricted and you know you own other ones, please do not continue accessing the restricted accounts you have and stick to playing on your original one. If you have trouble accessing or verifying it, please send us an email at [accounts@ppy.sh](mailto:accounts@ppy.sh) so that we can help you recover access to it.
```

Comment:
```suggestion
- **It is not your first account.** Multi-accounting is against the rules. Any accounts other than a person's first will become permanently restricted, but their original one will in most cases be left open and the user is encouraged to continue playing on it. If the multi-accounting is severe or abusive, a silence may be applied to their original account as a warning to cease creating new ones, and a restriction will be placed on the main account if the multi-accounting behaviour continues. If your account is restricted and you know you own other ones, please do not continue accessing the restricted accounts you have and stick to playing on your original one. If you have trouble accessing or verifying it, please send us an email at [accounts@ppy.sh](mailto:accounts@ppy.sh) so that we can help you recover access to it.
```

(`isn't` -> `is not`)